modules/docs/docgen/spec/docParser.spec.ts

```diff
@@ -590,14 +601,15 @@ describe('docParser', () => {
       expect(docs).toHaveProperty('0.type.value.typeParameters.1.value', 'string');
     });
 
-    it('should handle exported interfaces with an index signature', () => {
+    it.only('should handle exported interfaces with an index signature', () => {
```

Comment:
Ah, better take this `only` out.
```suggestion
    it('should handle exported interfaces with an index signature', () => {
```